tracing-okhttp3/src/main/java/com/palantir/remoting3/tracing/okhttp3/OkhttpTraceInterceptor.java

```diff
@@ -32,22 +29,10 @@
     @Override
     public Response intercept(Chain chain) throws IOException {
         Request request = chain.request();
-        OpenSpan span = Tracer.startSpan("remote call to redacted url", SpanType.CLIENT_OUTGOING);
-        Request.Builder tracedRequest = request.newBuilder()
-                .addHeader(TraceHttpHeaders.TRACE_ID, Tracer.getTraceId())
-                .addHeader(TraceHttpHeaders.SPAN_ID, span.getSpanId())
-                .addHeader(TraceHttpHeaders.IS_SAMPLED, Tracer.isTraceObservable() ? "1" : "0");
-        if (span.getParentSpanId().isPresent()) {
-            tracedRequest.header(TraceHttpHeaders.PARENT_SPAN_ID, span.getParentSpanId().get());
-        }
-
-        Response response;
-        try {
-            response = chain.proceed(tracedRequest.build());
-        } finally {
-            Tracer.completeSpan();
-        }
-
-        return response;
+        Request.Builder tracedRequest = request.newBuilder();
```

Comment:
This change means that we're losing the root span, don't think that's the desired behavior.

Reply:
To clarify, if the first server already had in an incoming request (which most would), that span would be the root span, so you would be losing the span between the root span the first span on the 2nd service?

Reply:
If the if the server has one: yes. But not all traces originate from servers.